src/app/admin/users/user-detail/user-detail.component.html

```diff
@@ -21,6 +21,10 @@ <h3>{{ "USERS.DETAIL.HEADLINE" | translate }}</h3>
             <strong>{{ "USERS.DETAIL.STATUS" | translate }}</strong
             >{{ user.active | activeDeactive }}
           </p>
+          <p *ngIf="user.expiresOn">
+            <strong>{{ "API-KEY.EXPIRES-ON" | translate }}</strong>
+            {{ (user.expiresOn) | dateOnly }}
```

Comment:
Can it be done so it doesnt always translate to english? If it's not easy then it might not matter